sympy/series/tests/test_limits.py

```diff
@@ -205,6 +209,20 @@ def test_doit():
     assert l.doit() == oo
 
 
+def test_Limits():
+    assert limit(sin(k) - sin(k + 1), k, oo) == Limits(-2, 2)
+    assert limit(cos(k) - cos(k + 1) + 1, k, oo) == Limits(-1, 3)
+
+    # not the exact bound
+    assert limit(sin(k) - sin(k)*cos(k), k, oo) == Limits(-2, 2)
+
+    #assert limit(tan(k)/3 - tan(k + 1), k, oo) == Limits(-oo, oo)
+
+    # test for issue #9934
+    assert limit(simplify(Sum(cos(n).rewrite(exp), (n, 0, k)).doit().rewrite(sin)), k, oo) == \
```

Comment:
@jksuom Can you please look why this is not working? The simplifcation of
```
>>> cos(1) + Limits(-3, 1)
<cos(1) - 3, cos(1) + 1>    # this works correctly
```

But this does not work
```
>>> limit(simplify(Sum(cos(n).rewrite(exp), (n, 0, k)).doit().rewrite(sin)), k, oo)
cos(1) + [-3, 1]
────────────────           # here the simplification of cos(1) + <-3, 1> does not occur.
2⋅(-1 + cos(1)) 
```